src/main/java/com/java/guesthouse/admin/ui/AdminController.java

```diff
@@ -0,0 +1,137 @@
+package com.java.guesthouse.admin.ui;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.servlet.ModelAndView;
+
+import com.java.guesthouse.admin.service.AdminService;
+import com.java.guesthouse.aop.HomeAspect;
+import com.java.guesthouse.member.service.dto.MemberDto;
+
+@Controller
+public class AdminController {
+
+    private final AdminService adminService;
+
+    public AdminController(AdminService adminService) {
+        this.adminService = adminService;
+    }
+
+    @RequestMapping(value = "/admin/memberList.do", method = RequestMethod.GET)
+    public ModelAndView memberList(HttpServletRequest request, HttpServletResponse response) {
+        HomeAspect.logger.info(HomeAspect.logMsg + "Admin memberManagement");
+        ModelAndView mav = new ModelAndView();
+        mav.addObject("request", request);
+
+        adminService.memberList(mav);
+        return mav;
+    }
+
+    @RequestMapping(value = "/admin/adminMemberRead.do", method = RequestMethod.GET)
+    public ModelAndView memberUpdate(HttpServletRequest request, HttpServletResponse response) {
```

Comment:
메서드명은 동사를 사용한다는 컨벤션을 지켜도 좋을 것 같습니다 :)
```suggestion
    public ModelAndView update(HttpServletRequest request, HttpServletResponse response) {
```
혹은
```suggestion
    public ModelAndView updateMember(HttpServletRequest request, HttpServletResponse response) {
```